pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -157,12 +172,19 @@ func (c *Cluster) makeDeployment(nodeName string, devices []rookalpha.Device, se
 			"--"},
 			commonArgs...)
 
-		var copyBinariesVolume v1.Volume
-		copyBinariesVolume, copyBinariesContainer = c.getCopyBinariesContainer()
-		// Add the volume to the spec and the mount to the daemon container
-		volumes = append(volumes, copyBinariesVolume)
-		volumeMounts = append(volumeMounts, copyBinariesContainer.VolumeMounts[0])
-
+	} else if osd.CephVolumeInitiated {
+		// if the osd was provisioned by ceph-volume, we need to launch it with rook as the parent process
+		command = []string{path.Join(rookBinariesMountPath, "tini")}
+		args = []string{
+			"--", path.Join(rookBinariesMountPath, "rook"),
+			"ceph", "osd", "start",
+			"--",
+			"--foreground",
+			"--id", osdID,
+			"--osd-uuid", osd.UUID,
+			"--conf", osd.Config,
+			"--cluster", "ceph",
```

Comment:
If `--cluster` is `ceph`, then I don't think we need it. It's like hardcoding the default :).

Reply:
i thought I tried without this option and it failed. Maybe it was something else. I'll try removing it again.

Reply:
So is the `--cluster` flag needed?